lib/ui/addon/templates/components/toolbar-search-field.hbs

```diff
@@ -1,11 +1,19 @@
-{{input value=value type="text" placeholder="Search"}}
+<div class={{concat "toolbar__search " @wrapperClass}}>
```

Comment:
Instead of `@wrapperClass` I think we can use normal `class`. Then I think if you pass `class="foo"` instead of `@wrapperClass="foo"` you should get both classes added to the element.
```suggestion
<div class="toolbar__search">
```

Reply:
I was torn on this. If you were the end-user of `<ToolbarSearchField />` would you think `class` would be applied to the `<input>`?

Reply:
I think I would look at the template before assuming, so I think it should be fine to use `class` directly, rather than introducing a new variable.

Reply:
Change made.